jvm/src/main/kotlin/app/cash/trifle/SignedData.kt

```diff
@@ -36,6 +36,9 @@ data class SignedData internal constructor(
     return isVerified
   }
 
+  fun verifyAndExtract(certAnchor: Certificate): VerifiedData? =
```

Comment:
should we replace `verify` with this method or reduce visibility of `verify` method?

Reply:
considered that, but decided to keep it for now to not break current clients, but also because I could see a service wanting to verify before passing along potentially as a use-case, so figured I'd leave it in for now and see how we feel about it going forward.